### DISCUSSION TITLE
get all of the matrix value after running plotEmbedding

### DISCUSSION THREAD

**Discussion by tongyt1225** [GreenleafLab/ArchR#1198] [Questions / Documentation]
Hi，I want to get  all of the matrix value  after running plotEmbedding, but only part of the matrix value in .log file.  Is there any way to get them all？Thanks a lot.
![截屏2021-11-30 上午11 16 03](https://user-images.githubusercontent.com/52367576/143979617-d2a93f33-6250-4f42-95e5-d4e6ab61d4b9.png)

**Comment by rcorces** [comment]
As indicated in the instructions, this does not belong in the Issues section.
![image](https://user-images.githubusercontent.com/10343046/144077527-1a87b01f-d6f8-468e-a592-3b344f5d2003.png)

This is a usage question and not an error or bug report. In the future please post to Discussions. I am transferring your post there.

**Comment by rcorces** [comment]
`plotEmbedding()` returns a plot, not a matrix. use `getMatrixFromProject()`. Please look at all available functions before posting usage questions https://www.archrproject.com/reference/index.html
